src/components/Discussions/DiscussionCard.tsx

```diff
@@ -388,11 +388,15 @@ export const DiscussionCard = (props: IProps) => {
   };
 
   useEffect(() => {
-    if ((onAddComplete !== undefined) !== editing) {
+    if (Boolean(onAddComplete) !== editing) {
       handleReset();
       setEditing(onAddComplete !== undefined);
       if (!onAddComplete) setEditSubject(discussion.attributes?.subject);
     }
+    if (Boolean(onAddComplete) && Boolean(discussion.attributes?.subject)) {
+      setEditSubject(discussion.attributes?.subject);
+      setMyChanged(true);
```

Comment:
Do we need to set toolChanged here?  Do we want this to be considered "changed" if we move off the step without saving this dicussion?  I can see arguments both ways...